temporal-sdk/src/main/java/io/temporal/workflow/WorkflowInfo.java

```diff
@@ -47,6 +47,7 @@
    */
   long getRunStartedTimestampMillis();
 
+  @Deprecated
```

Comment:
Please add a comment on why it was deprecated and which method should be used instead.